package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -214,8 +214,8 @@ class HydrogenBondAnalysis_OtherFF(HydrogenBondAnalysis):
 .. _`10.1002/prot.340090204`: http://dx.doi.org/10.1002/prot.340090204
 
 
-Example
```

Comment:
Just fyi: with numpy docs, never use `Example` or `Examples` in a normal section context. sphinx napoleon rewrites it. So you have to ne a bit creative with labelling sections in the main part of the page.